### ISSUE TITLE
Wrong accent color, Topbar

### ISSUE THREAD

**Issue by GiornoRetr0** [vinceliuice/WhiteSur-gtk-theme#1057]
accent color of shell apps(f.e files) doesn't match with the theme selected(in this case WhiteSur dark). Also tray and top bar don't seem to be changed at all after applying the theme.

![Pasted image](https://github.com/vinceliuice/WhiteSur-gtk-theme/assets/77203164/c73cc3e0-cada-429a-a908-e1a62e84938e)

**Comment by GiornoRetr0** [comment]
> You need use gnome-tweaks or theme extension setting to change shell theme

I already did that, the problem still persists

**Comment by vinceliuice** [comment]
Try the last one

**Comment by Sam77repo** [comment]
I have the last update and this issue persist

**Comment by Sam77repo** [comment]
> > You need use gnome-tweaks or theme extension setting to change shell theme
> 
> I already did that, the problem still persists

YES I FIXED! If you don't mind not being able to use the yellow theme anymore, go ahead and do it: 

Open (Personal Folder > WhiteSur-gtk-theme > shell) and open with text editor the lib-core.sh

 press Ctrl + F and write Yellow

 in Decoration delete the line with export c_yellow="\033[1;32m".

 And reinstal theme on Terminal, following this steps:

 go to cd  WhiteSur-gtk-theme/    

Run this command ./install.sh

And
(if you wanna other color change -t for the color of your preference)
./install.sh -m -t blue -l -N stable --normal --round

restart your computer and enjoy :)
